src/components/FormInput.vue

```diff
@@ -9,7 +9,7 @@
 <script>
 export default {
   name: "FormInput",
-  props: ["title", "model"],
+  props: ["title", "bind"],
```

Comment:
I think I wrote this, but not really sure what I was thinking - this is a bad name.